### ISSUE TITLE
rucio --help and rucio --version fail if no config file can be found

### ISSUE THREAD

**Issue by maxnoe** [rucio/rucio#6460]
### Description

Running `rucio --help` fails with an exception if no config file can be found.

This behavior is very bad, since `--help` would show the `--config` option one has to use to point to a config file.

Please make `--help` work without a config file.

### Steps to reproduce

Run `rucio --help` or `rucio --version` when no config file is present in the standard search paths:

```
(venv) [root@142b262de71d /]# rucio --help
Traceback (most recent call last):
  File "/venv/bin/rucio", line 52, in <module>
    from rucio.rse.protocols.protocol import RSEProtocol
  File "/venv/lib64/python3.9/site-packages/rucio/rse/protocols/protocol.py", line 262, in <module>
    RSEDeterministicTranslation._module_init_()  # pylint: disable=protected-access
  File "/venv/lib64/python3.9/site-packages/rucio/rse/protocols/protocol.py", line 231, in _module_init_
    policy_module = config.config_get('policy', 'lfn2pfn_module')
  File "/venv/lib64/python3.9/site-packages/rucio/common/config.py", line 213, in config_get
    raise err
  File "/venv/lib64/python3.9/site-packages/rucio/common/config.py", line 191, in config_get
    return convert_type_fnc(get_config().get(section, option))
  File "/venv/lib64/python3.9/site-packages/rucio/common/config.py", line 765, in get_config
    __CONFIG = Config()
  File "/venv/lib64/python3.9/site-packages/rucio/common/config.py", line 789, in __init__
    raise RuntimeError('Could not load Rucio configuration file. '
RuntimeError: Could not load Rucio configuration file. Rucio looked in the following paths for a configuration file, in order:
	/venv/etc/rucio.cfg
	/opt/rucio/etc/rucio.cfg
```

### Rucio Version

rucio-clients 33.2.0

**Comment by bari12** [comment]
Hmm, I think this is a regression, since this was initially fixed in #4601

**Comment by maxnoe** [comment]
This issues seems to have been fixed in rucio 33.5 and above
